app/forms/schools/add_participants/base_wizard.rb

```diff
@@ -493,7 +493,7 @@ def cohort_for_ect_with_induction_start_date
       end
 
       def cohort_for_transfer
```

Comment:
I assume this is only used when it's a "frozen" transfers?  The naming isn't clear to me so I'm not sure.